xroad-catalog-lister/README.md

```diff
@@ -12,32 +12,57 @@ See also the [Installation Guide](../doc/xroad_catalog_installation_guide.md) an
-| [spring.jpa.show-sql](https://docs.spring.io/spring-boot/appendix/application-properties/index.html#application-properties.data.spring.jpa.show-sql)                                 | N        | `false`                                   | Use to debug SQL statements executed. Preferably `false` in production.                                                                                                                                                                | 1.0.0 |
-| [spring.jpa.hibernate.ddl-auto](https://docs.spring.io/spring-boot/appendix/application-properties/index.html#application-properties.data.spring.jpa.hibernate.ddl-auto)             | Y        | `none`                                    | Keep to `none` to prevent hiberante from trying to update the database.                                                                                                                                                                | 1.0.0 |
-| spring.jpa.properties.hibernate.dialect                                                                                                                                              | Y        | `org.hibernate.dialect.PostgreSQLDialect` | PostgreSQL is the supported RDMBS. But change to other dialect according to the used RDBMS, if needed. Available options can be found [here](https://docs.jboss.org/hibernate/stable/orm/javadocs/org/hibernate/dialect/Dialect.html). | 1.0.0 |
+#### Mandatory configurations required by common features
 
-#### OpenAPI documentation configurations
+| Parameter                          | Required | Defaults | Description                                                                         | Since |
+|------------------------------------|----------|----------|-------------------------------------------------------------------------------------|-------|
+| `xroad-catalog.shared-params-file` | Y        |          | A parameter for setting the path to shared params file exported from X-Road server. | 1.0.0 |
+
+#### Mandatory configurations required by Finland-specific features
+
+| Configurations                     | Required | Defaults | Description                                                                                                           | Since |
+|------------------------------------|----------|----------|-----------------------------------------------------------------------------------------------------------------------|-------|
+| `xroad-catalog.country.fi.enabled` | Y        | `false`  | A parameter to enable/disable Finland specific features. If `false`, then all Finland specific endpoints are disabled | 1.0.0 |
```

Comment:
Same here, I think it could be argued that this is not required as `false` is a reasonable default.